fuzzers/qemu_systemmode/src/fuzzer_breakpoint.rs

```diff
@@ -30,12 +30,11 @@ use libafl_bolts::{
 use libafl_qemu::{
     breakpoint::Breakpoint,
     command::{EndCommand, StartCommand, StdCommandManager},
-    edges::{edges_map_mut_ptr, QemuEdgeCoverageHelper, EDGES_MAP_SIZE_IN_USE, MAX_EDGES_FOUND},
+    edges::{edges_map_mut_ptr, QemuEdgeCoverageTool, EDGES_MAP_SIZE_IN_USE, MAX_EDGES_FOUND},
```

Comment:
Frida uses `CoverageRuntime`

Reply:
(and it's not really a tool, is it?)

Reply:
Maybe `Plugin`/`Extension`?

Reply:
`Plugin` collides with qemu plugins, it could be misunderstood no?

Reply:
We should keep the names for the same usecase the same between Frida and QEMU though, I think

Reply:
We can also change the frida name I guess

Reply:
Maybe ask ChatGPT :D